database/seeds/SponsorsSeeder.php

```diff
@@ -42,6 +42,10 @@ public function run()
 
         // 4MAY20-VC1-CH1P-HA-012020 is in sponsor 5, unmodified
         // 1MAY20a-VC2-CH1-HI-042019 is in sponsor 5, unmodified
+
+        $noTapSponsor = factory(App\Sponsor::class)->create(['name' => "No Tap Project", "can_tap" => false]);
+        $noTapSponsor->evaluations()->saveMany($this->qualifyPrimarySchoolers());
+        $noTapSponsor->evaluations()->saveMany($this->veryfiesKids());
```

Comment:
These last two adds rules to
- force primary school kids to be qualified by a younger kid
- force users to click a box to verify they have documentation

Not strictly necessary for this, we'll see if it testing spots it ;-)

Reply:
Should I remove these?